### ISSUE TITLE
Bash autocompletion

### ISSUE THREAD

**Issue by s-hemer** [intel/bmap-tools#118]
I started a bash-completion script for bmaptool. It is still a bit hacky not yet ready for a pull rq but what I'd like to know: where to put that file, packaging/ ?

**Comment by dedekind** [comment]
Hi, first quick question - why script instead of argcomplete module, which will automatically add all options and commands?

**Comment by s-hemer** [comment]
Good point. Indeed, that is potentially the better way to go when touching the Python scripts directly, yet I just wanted to add that functionality to my package system install, so I went the way with the completion script that I was more familiar with.

**Comment by dedekind** [comment]
There are many examples googlable, but here is a project I currently actively maintain which uses argpcompete. Works well fine: https://github.com/intel/pepc . However, the probject may be too complex to serve as an example. But FWIW.

**Comment by s-hemer** [comment]
I am too busy with other things, yet if someone is interested: https://gist.github.com/s-hemer/760f13ced6f4f6d99b0cddde5757470f

**Comment by dedekind** [comment]
Hello, the project was moved and now owned by a new team. Please, work with them. Check the README.md file. Thanks!
